src/components/DatePickerDocked/utils.ts

```diff
@@ -0,0 +1,114 @@
+import type { TextStyle, ViewStyle } from "react-native";
+import type { ComponentStylePropWithVariants } from "src/types";
+
+export const MONTHS_DATA = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"]
+
+type States = '';
+
+type CustomProps = {
+    roundness?: string;
+    headerBackgroundColor?: string;
+
+    container?: ViewStyle;
+};
+
+export const datePickerDockedStyles: ComponentStylePropWithVariants<TextStyle, States, CustomProps> = {
+    roundness: 'shapes.corner.extraSmall',
+    headerBackgroundColor: 'colors.surface',
+
+    container: {
+        minHeight: 360,
+        minWidth: 360,
+    },
+};
+
+type DatePickerHeaderCustomProps = {
+    datePickerHeader: ViewStyle;
+    buttonContainer: ViewStyle;
+    buttonWrapper: ViewStyle;
+    spacer: ViewStyle;
+    yearButtonStyle?: ViewStyle;
+    yearInnerStyle?: ViewStyle;
+    yearLabelStyle?: TextStyle & { typescale?: string };
+    daysWrapperStyle?: ViewStyle;
+};
+
+export const datePickerDockedHeaderStyles: ComponentStylePropWithVariants<
+    TextStyle,
+    '',
+    DatePickerHeaderCustomProps
+> = {
+    datePickerHeader: {
+        position: 'absolute',
+        top: 0,
+        right: 0,
+        left: 0,
+        zIndex: 10,
+    },
+    buttonContainer: {
+        height: 46,
+        marginTop: 'spacings.2',
+        marginBottom: 'spacings.2',
+        marginRight: 'spacings.4',
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    buttonWrapper: {},
+    spacer: { flex: 1 },
+    yearLabelStyle: {
+        typescale: 'typescale.bodyMedium',
+        opacity: 0.7,
+    },
+    yearButtonStyle: {
+        alignSelf: 'flex-start',
+        marginLeft: 6,
+        borderRadius: 'shapes.corner.extraSmall' as unknown as number,
+    },
+    yearInnerStyle: {
+        paddingLeft: 'spacings.4',
+        flexDirection: 'row',
+        alignItems: 'center',
+        borderRadius: 'shapes.corner.extraSmall' as unknown as number,
+    },
+    daysWrapperStyle: {
+        marginHorizontal: 'spacings.3'
+    }
+};
+
+type DatePickerMonthPickerCustomProps = {
+    month?: TextStyle;
+    selectedMonth?: TextStyle;
+    monthButton?: ViewStyle;
+    monthInner?: ViewStyle;
+    selectedMonthInner?: ViewStyle;
+    monthLabel?: TextStyle;
+};
+
+export const datePickerMonthPickerStyles: ComponentStylePropWithVariants<
+    TextStyle,
+    '',
+    DatePickerMonthPickerCustomProps
+> = {
+    backgroundColor: 'colors.surface',
+
+    month: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'flex-start'
+    },
+    selectedMonth: { color: 'colors.onPrimary' },
+    monthButton: {
+        width: '100%',
+        overflow: 'hidden',
+    },
+    monthInner: {
+        height: 46,
+        alignItems: 'center',
+        justifyContent: 'flex-start',
+        flexDirection: 'row',
+    },
+    selectedMonthInner: { backgroundColor: 'colors.primary' },
```

Comment:
Incorrect background color. Please check the m3 guidlines

Reply:
@jasimawan there are more specific color guidelines on the same link.
https://m3.material.io/components/date-pickers/specs#940dc547-8558-496a-b4ae-6d521b96ffc2

In your reference there are two parts:
1. Container (attributed to the number 8)
2. Selected list item (attributed to the number 6)

What you are referring to is the Container Element styles which apparently somehow is the overall background color of the menu container.
What we should be using is the color for Selected list item which unfortunately is not mentioned in the list you shared. For that you'll have to refer to the link I provided.

Reply:
resolved @shashank1010 @thetaungg

Reply:
I don't see the changes. ;)
Not yet resolved :D

Reply:
I just pushed the changes, now working on your changes @shashank1010